### DISCUSSION TITLE
All romanian grammar explained in german

### DISCUSSION THREAD

**Discussion by major-tweaks** [LearnRomanian/romanian#4] [Ideas]
Dear all,
I found a nice website containing a lot of explanations, if not all, about the romanian grammar, in german:
https://www.grammatiken.de/rumaenische-grammatik-online-lernen/
The site is free of charge and no account needed, of course, one can sign up to a paid course which will build up on the lecture.

@vxern  Maybe to be added to the list? 
BR, Major-Tweaks

**Comment by vxern** [comment]
Hi, @major-tweaks, thanks a lot for the suggestion, looks like a really useful resource for German speakers. Added.
